src/client/utils.ts

```diff
@@ -3,6 +3,7 @@ import fs from 'fs';
 
 import { DEFAULT_SERVER_PORT } from '../constants';
 import { IGameState, IJoinRoomOpts, ISimpleGameState } from '../types';
+import { boolean } from '@colyseus/schema/lib/encoding/decode';
```

Comment:
I think that this import has been automatically added by the IDE and it is not necessary

Reply:
Removed